conductor/clarisse/scripted_class/submission.py

```diff
@@ -48,6 +55,37 @@
 from conductor.lib import conductor_submit
 from conductor.native.lib.data_block import ConductorDataBlock
 from conductor.native.lib.gpath import Path
+from conductor.native.lib.gpath_list import PathList
+
+PROJECT_EXTENSION_REGEX = r"(\.ct\.project|\.project)"
+CT_PROJECT_EXTENSION = ".ct.project"
+
+
+def _get_path_line_regex():
+    """
+    Generate a regex to help identify filepath attributes.
+
+    As we scan project files to replace windows paths, we use this regex which
+    will be something like: r'\s+(?:filename|filename_sys|save_as)\s+"(.*)"\s+' 
+    only longer.
+    """
+    classes = ix.application.get_factory().get_classes()
+    file_attrs = []
+    for klass in classes.get_classes():
+        attr_count = klass.get_attribute_count()
+        for i in range(attr_count):
+            attr = klass.get_attribute(i)
+            hint = attr.get_visual_hint()
+            if hint in [
```

Comment:
Extract the hardcoded list into a tuple outside of the loops.

Reply:
done